apps/dot-voting/app/store/votes.js

```diff
@@ -91,24 +93,28 @@ const loadVoteDataAllocation = async (vote, voteId) => {
           canExecute,
           options,
         }
-        // const symbol
-        // const tokenAddress = '0x' + vote.executionScript.slice(794, 834)
-        // if (tokenAddress === ETHER_TOKEN_FAKE_ADDRESS) {
-        //   symbol = 'ETH'
-        // }
-        // else {
-        //   symbol = await getTokenSymbol(app, tokenAddress)
-        // }
+        const allocationsAddress = '0x' + vote.executionScript.slice(10,50)
+        const allocationsAccountId = parseInt(vote.executionScript.slice(514, 578), 16).toString() //should be 514, 578
```

Comment:
The comment seems to repeat the code itself. An explanation of what these numbers mean might be nice. Better yet, rather than explain them with a comment, explain them by declaring them as well-named constants.